LttPAdjuster.py

```diff
@@ -29,7 +29,8 @@
 
 
 GAME_ALTTP = "A Link to the Past"
-
+WINDOW_MIN_HEIGHT = 425
```

Comment:
Sorry about the extra commit here (and I forgot to squash before merging). I mixed up the constant names and had height and width reversed (but with the correct value in the minsize() call). The extra commit fixes that naming problem